src/main/cms/cms_menu_ledstrip.c

```diff
@@ -39,51 +39,88 @@
 
 #include "fc/config.h"
 
+#include "io/ledstrip.h"
+
+#include "interface/settings.h"
 
 #ifdef USE_LED_STRIP
 
 static bool featureRead = false;
 static uint8_t cmsx_FeatureLedstrip;
+#ifdef USE_LED_PROFILES
```

Comment:
Not quite sure if we need `USE_LED_PROFILES` at all. If `USE_LED_STRIP` was defined, but not `USE_LED_PROFILES`, what could the user do with the LED_STRIP, and how would they configure it?

Reply:
@mikeller The idea with USE_LED_PROFILES is to cater for targets that will run out of space if this feature is added, i.e. they can still use the LED_STRIP functionality as is now.  I agree we may not need it anyway since LED_STRIP will probably be removed then for other more important features.  I think the cases covered by USE_LED_PROFILES  can be handled by USE_LED_STRIP where necessary.

Reply:
@mikeller Ok, I understand.  If we aren't planning to use the 'legacy mode only' version it's probably better to remove it.  I'll have a look at removing it so only USE_LED_STRIP & USE_LED_STRIP_STATUS_MODE is used.

Reply:
Done.